doc/reference/reference_lua/box_space/id.rst

```diff
@@ -0,0 +1,25 @@
+.. _box_space-id:
+
+===============================================================================
+space_object:id
```

Comment:
This should have a dot: `space_object.id`

Reply:
fixed in 22a984c